src/FirstSteps/RPG/Game.cs

```diff
@@ -67,21 +66,30 @@ private static string MenuSelect()
         public static void GameMenu()
         {
             AnsiConsole.MarkupLine($"{Emoji.Known.BowAndArrow} [darkgreen]Welcome to the RPG game [/] {Emoji.Known.CrossedSwords}");
-
-            Game.CreateHero();
+            var confirmation = new ConfirmationPrompt("Do you want to load the hero from file?");
+            var answer = AnsiConsole.Prompt(confirmation);
+            if(answer)
+            {
+                
+                HeroesRepository.LoadHero();
```

Comment:
Here you should have something like `_hero = HeroesCreator.LoadFromFile();` and inside that new method you should use the repository to load `HeroModel` and then create a 'Hero` object based on the model.

Reply:
`HeroesCreator.cs` is responsible for creating a hero. So far that class had one method `public static Hero Create(string name, Races race)` that was used to create a new hero object. 

Now we want to add an option that allows us to create a hero in another way. We want to load it from the JSON file. So naturally `HeroesCreator.cs` is the best place to add such a function. 

It could look like this:
```csharp
public static Hero LoadFromFile()
{
    //load hero from file and assign it to hero model object (use repository)
    
    //check the race of the loaded hero model and choose a proper class
    //return concrete hero object by the new constructor (Human etc.) 
}
```